controller/workitemtype.go

```diff
@@ -16,25 +16,39 @@ const (
 // WorkitemtypeController implements the workitemtype resource.
```

Comment:
The Create method should at least set the Headers. Tho a conditional response won't much sense here. I guess we could reuse the Conditional function and just provide Last-Modifed/Etags etc that on't match anything the client would know anyway.